stingray/bispectrum.py

```diff
@@ -78,6 +79,23 @@ def compute_bispectrum(self, lc):
         bispec = scipy.fftpack.fftshift(bispec) / self.k
         return bispec
 
+    def plot(self):
+        bispectrum = self.compute_bispectrum(self.lc)
+        print bispectrum
+        if self.nfft % 2 == 0:
+            freaxis = np.transpose(
+                np.arange(-1 * self.nfft / 2, self.nfft / 2))
+        else:
+            freaxis = np.transpose(
+                np.arange(-1 * (self.nfft - 1) / 2, (self.nfft - 1) / 2 + 1))
+        cont = plt.contourf(freaxis, freaxis, abs(
+                    bispectrum), 200, cmap=plt.cm.Spectral_r)
+        plt.colorbar(cont)
+        plt.title('Bispectrum ')
+        plt.xlabel('f1')
+        plt.ylabel('f2')
+        plt.show()
```

Comment:
Hmm... a function to draw and show in one run. There are both pros and cons of this. The good thing is that users on Python interpreter will not have to execute `plt.show()` by themselves for viewing the plot. While the complication is in Jupyter Notebooks. I have seen that the plots are mostly preferred to be inline and no pop-ups are expected from matplotlib. Since we plan to keep a good amount of example notebooks, I think we should not keep this.